### PR TITLE
Adding snippet links for missing person class.

### PR DESCRIPTION
**Pull request by JRAlexander**
Adding snippet links for missing person class.

Fixes dotnet/docs [#6106](https://github.com/dotnet/docs/issues/6106).

### CODE REVIEW

**Comment by JRAlexander** [comment]
Closing and reopening

**Comment by JRAlexander** [comment]
Closing and reopening

File: xml/System.Windows.Controls/TabControl.xml [hunk 1]
```diff
@@ -289,9 +289,13 @@
  The following example uses the <xref:System.Windows.Controls.TabControl.ContentTemplateSelector%2A> property to display the content of a <xref:System.Windows.Controls.TabItem> differently, depending on the hometown of the selected person.  
   
  [!code-xaml[TabControlContentTemplateSelector#1](~/samples/snippets/csharp/VS_Snippets_Wpf/TabControlContentTemplateSelector/CSharp/Window1.xaml#1)]
```
Comment (rpetrusha) [inline xml/System.Windows.Controls/TabControl.xml]: This separation of two XAML fragments seems confusing. I'd consider combining them for both C# and VB.

**Comment by JRAlexander** [comment]
Closing and reopening to trigger build

**Pull request merged**